pyaedt/modeler/stackup_3d.py

```diff
@@ -0,0 +1,2284 @@
+        axis="X",
+        reference_system=None,
+        frequency=1e9,
+    ):
+        """Create a new line.
+
+        Parameters
+        ----------
+        line_width : float
+            Line width. It can be the physical value or the line impedance.
+        line_length : float
+            Line length. It can be the phisical length or the electrical length.
+        is_electrical_length : bool, optional
+            Either if the line_length is an electrical length or physical one.
+        is_impedance : bool, optional
+            Either if the line_width is an an impedance or geometrical value.
+        line_position_x : float, optional
+            Line Center start x position.
+        line_position_y : float, optional
+            Line Center start y position.
+        line_name : str, optional
+            Line name.
+        axis : str, optional
+            Line orientation axis.
+        reference_system : str, optional
+            Line Reference System. If `None` a new coordinate system will be created.
+        frequency : float, optional
+            Frequency value for line calculation in Hz.
+
+        Returns
+        -------
+        :class:`pyaedt.modeler.stackup_3d.Line`
+        """
+        if not line_name:
+            line_name = generate_unique_name("{0}_line".format(self._name), n=3)
+        dielectric_layer = None
+        for v in list(self._stackup._stackup.values()):
+            if v._index == self._index - 1:
+                dielectric_layer = v
+                break
+        if dielectric_layer is None:
+            self._app.logger.error("There is no layer under this layer")
+
+        created_line = Line(
+            self._app,
+            frequency,
+            line_width if is_impedance else None,
+            line_width if not is_impedance else None,
+            self,
+            dielectric_layer,
+            line_length=line_length if not is_electrical_length else None,
+            line_electrical_length=line_length if is_electrical_length else None,
+            line_position_x=line_position_x,
+            line_position_y=line_position_y,
+            line_name=line_name,
+            reference_system=reference_system,
+            axis=axis,
+        )
+        created_line.aedt_object.group_name = "Layer_{}".format(self._name)
+        self._obj_3d.append(created_line.aedt_object)
+        self._stackup._object_list.append(created_line)
+        return created_line
+
+    @pyaedt_function_handler()
+    def polygon(self, points, material="copper", is_void=False, poly_name=None):
+        """Create a new Polygon.
+
+        Parameters
+        ----------
+        points : List
+            Points list of [x,y] coordinates.
+        material : str, optional
+            Material name.
+        is_void : bool, optional
+            Either if the polygon is a void or not.
+            On ground layers it will act opposite as the boolean value since ground is negative.
```

Comment:
```suggestion
            Whether the polygon is a void. The default is ``False``.
            On ground layers, it will act opposite of the Boolean value because the ground
            is negative.
```